### ISSUE TITLE
problem with starting use weka wrapper 

### ISSUE THREAD

**Issue by edess** [fracpete/python-weka-wrapper#22]
Hello Fracpete, 
Hope you're fine! 

I'm new in machine learning system, and know trying to implement one ! 

I succeeded bulding a classifier for images' classification, following a tutorial and adding many things to make it run. And now I want to evaluate the classifier by plotting the confusion matrix. 
I read a lot about confusion matrix and now understand all the theories behind! 

The classifier gives me some files (like X.svm.range, X.svm.model, X.svm.scale and a codebook). Actually my classifier uses Bag of visual word with sift.   

My problem is I don't know which of this file, I should use to start my evaluation. 
I know I have to split my dataset into training set and test set (like the iris dataset, used in almost all examples ). I also install python-weka-wrapper (I'm using python 2.7), but don't now from where to start! 

Do you have any clue or any link that could help me from the very beginning?
just let me know if you need some more precisions. 

Thanks! 
Edess
